### PR TITLE
[OBMIN-357] redesing footer

### PR DESCRIPTION
**Pull request by Voloshkina**

### CODE REVIEW

File: src/components/footer/Footer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Footer } from '.';
+
```
Comment (Wolshebnik) [inline src/components/footer/Footer.stories.tsx:4]: remove space

File: src/config/theme/colors.ts [hunk 2]
```diff
@@ -272,6 +272,11 @@ const searchResult = Object.freeze({
   gradient: '100.18deg, #8fd3e1 7.33%, #51b2d4 97.87%',
 });
 
+const footerColors = Object.freeze({
+  white,
+  colorText: ' rgba(255, 255, 255, 0.5)',
```
Comment (Wolshebnik) [inline src/config/theme/colors.ts:277]: an empty string in the start
should it be?

File: src/components/footer/styles.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  > svg {
+    width: 20px;
+    height: 20px;
+  }
+
+  ${({ theme }) =>
+    !theme.responsive.isMobile &&
+    css`
+      margin-bottom: 44px;
+      > svg {
```
Comment (Wolshebnik) [inline src/components/footer/styles.ts:92]: space

File: src/components/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            <Deals background heartIcon to={toDeals} text={text} />
+          </Styles.Lists>
+
+          <Styles.Lists>
+            <Styles.Span>
+              <Icon.Phone />
+            </Styles.Span>
+
+            <Styles.Contact href="mailto:&#111;&#098;&#109;&#105;&#110;&#121;&#097;&#115;&#104;&#107;&#097;&#046;&#115;&#112;&#097;&#099;&#101;&#064;&#103;&#109;&#097;&#105;&#108;&#046;&#099;&#111;&#109;">
+              &#111;&#098;&#109;&#105;&#110;&#121;&#097;&#115;&#104;&#107;&#097;&#046;&#115;&#112;&#097;&#099;&#101;&#064;&#103;&#109;&#097;&#105;&#108;&#046;&#099;&#111;&#109;
```
Comment (Wolshebnik) [inline src/components/footer/index.tsx:44]: do this text as props

File: src/components/footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            </Styles.Contact>
+
+            <Styles.Contact href="tel:&#43;&#51;&#56;&#48;&#57;&#51;&#49;&#50;&#51;&#52;&#53;&#54;&#55;">
+              +3 80 (93) 123 45 67
```
Comment (Wolshebnik) [inline src/components/footer/index.tsx:48]: the same

File: src/components/footer/index.tsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            </Styles.Contact>
+
+            <Styles.Contact href="tel:&#43;&#51;&#56;&#48;&#57;&#51;&#49;&#50;&#51;&#52;&#53;&#54;&#55;">
+              +3 80 (93) 123 45 67
```
Comment (Wolshebnik) [inline src/components/footer/index.tsx:52]: and this

File: src/components/logo/styles.ts [hunk 7]
```diff
@@ -37,6 +35,8 @@ export const ImgWrapper = styled.div<{ inFooter?: boolean }>`
 `;
 
 export const ProjectName = styled.p<{ inFooter?: boolean }>`
+  font-family: 'Expletus Sans', cursive;
+  font-weight: 400;
```
Comment (cloudmenson) [inline src/components/logo/styles.ts:39]: 'Expletus Sans' without quotes

**Review by Kyslytsia — APPROVED** [review]

**Pull request merged**